application/src/databaseengine.cpp

```diff
@@ -0,0 +1,94 @@
+#include "databaseengine.h"
+
+#include <QSql>
+#include <QSqlQuery>
+#include <QFile>
+#include <QDir>
+#include <QDebug>
+#include <QString>
+
+static const QString DATABASE_PATH = QDir::homePath() + "/Database";
+
+DatabaseEngine::DatabaseEngine()
+{
+
+}
+
+void DatabaseEngine::createDatabase(const User &user)
+{
+    if (!m_database.isOpen()) {
+        QString filePath = DATABASE_PATH + "/" + user.id + ".db";
+        QDir way(QDir::homePath());
+        way.mkdir("Database");
+        QFile file(filePath);
+        file.open(QIODevice::WriteOnly);
+        openDatabase(user);
+        if (createTableInDatabase()) {
+            if (insertIntoDatabase(user)) {
+                file.close();
+            }
+        }
+        file.close();
+    } else qDebug() << "База данных открыта";
```

Comment:
если начал условие с блоком фигурных скобок {} то все его составляющие должны заключаться в {}
```suggestion
    } else {
        qDebug() << "База данных открыта";
    }        
```